app/components/Layout/Header.tsx

```diff
@@ -134,12 +134,19 @@ const HeaderLayout: React.FunctionComponent<Props> = ({
             ) : (
               <>
                 <li>
-                  <LoginButton>
+                  <LoginButton idpHint="idir">
                     <button className="nav-button" type="submit">
                       Login (IDIR)
                     </button>
                   </LoginButton>
                 </li>
+                <li>
+                  <LoginButton idpHint="bceidboth">
+                    <button className="nav-button" type="submit">
+                      Login (BCeID)
```

Comment:
We might want to rename this to something like "Industrial Operator Login (BCeID)" and "Program Administrator Login (IDIR)" to avoid confusion